### PR TITLE
DataMigration.md edits

### PR DESCRIPTION
**Pull request by anTrodaire**
As an assignment for a technical writing course, we were told to edit documentation on projects on GitHub

### CODE REVIEW

File: docs/DataMigration.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ myDatabase.collection({
 
 ## autoMigrate
 
-By default, the migration automatically performs when the collection is created. If you have much data or the migrationStrategies take very long, it could be better to start the migration 'by hand' and show the migration-state to the user as loading-bar.
+By default, the migration automatically happens when the collection is created. If you have lots of data or the migrationStrategies take long a long time, it might be better to start the migration 'by hand' and show the migration-state to the user as a loading-bar.
```
Comment (pubkey) [inline docs/DataMigration.md]: `long a long time` - is this correct?

File: docs/DataMigration.md [hunk 2]
```diff
@@ -123,15 +123,15 @@ migrationState$.subscribe(
...
+If you don't want to show the state to the user, you can also use `.migratePromise()`:
 ```js
   const migrationPromise = messageCol.migratePromise(10);
   await migratePromise;
 ```
 
 
 ## hint
-If your migration takes very long, combine it with the leaderElection to make sure you not waste your users ressources by running it in 2 open tabs.
+If your migration takes long a long time, combine it with the leaderElection to make sure you don't waste your users' resources by running it in 2 open tabs.
```
Comment (pubkey) [inline docs/DataMigration.md]: `long a long time` - is this correct?

**Comment by pubkey** [comment]
@anTrodaire Thank you for the great work. I added to comments where I think something might be wrong.

File: docs/DataMigration.md (hunk 1 above)
Comment (anTrodaire) [inline docs/DataMigration.md]: good catch, thanks

File: docs/DataMigration.md (hunk 2 above)
Comment (anTrodaire) [inline docs/DataMigration.md]: good catch, thanks

**Comment by anTrodaire** [comment]
Sorry about that, the mistakes have been fixed and the changes pushed.

**Pull request merged**